yocto/recipes-conectivity/ola/ola_git.bb

```diff
@@ -0,0 +1,73 @@
+
+LICENSE = "GPLv2 & LGPLv2.1"
+LIC_FILES_CHKSUM = "file://LICENCE;md5=7aa5f01584d845ad733abfa9f5cad2a1"
+
+#DEPENDS_class-nativesdk = "bison libusb1 protobuf protobuf-native pkgconfig-native ossp-uuid ola-native "
+DEPENDS_class-target    = "bison libmicrohttpd avahi libusb1 libftdi cppunit protobuf protobuf-native ola-native"
+DEPENDS_class-native    = "protobuf bison-native flex-native pkgconfig-native ossp-uuid-native"
+
+PV = "0.10.6"
+
+# Esta revision de git tiene un parche para protobuf, la tag de 0.10.6 no compila.
+SRCREV = "00dc86a48ec4c528cec90166435b440f283a9c86" 
+SRC_URI = "git://github.com/OpenLightingProject/ola.git;protocol=https"
+
+S = "${WORKDIR}/git"
+B = "${WORKDIR}/git"
+
+inherit autotools-brokensep relative_symlinks
+# autotools-brokensep 
+# forces in-source building (ola has bugs for out-of-tree builds). 
+# See: https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#migration-1.7-autotools-class-changes
+# 
+# relative_symlinks
+# Absolute symbolic links (symlinks) within staged files are no longer permitted and now trigger an error.
+# inherit relative_symlinks within the recipe to turn those absolute symlinks into relative
+# https://www.yoctoproject.org/docs/2.4/ref-manual/ref-manual.html#migration-2.3-absolute-symlinks
+
+#EXTRA_OECONF = " --disable-unittests --disable-rdm-tests --disable-python-libs --disable-java-libs --disable-all-plugins"
+EXTRA_OECONF  = " --disable-unittests --disable-rdm-tests --disable-java-libs "
+
+# nativesdk is for the toolchain, only libola and ola_common are required, all plugins are dynamic
+EXTRA_OECONF_append_class-nativesdk = " --with-ola-protoc-plugin=${STAGING_BINDIR_NATIVE}/ola_protoc_plugin \
+                                        --disable-all-plugins --disable-e133 --disable-python-libs --disable-examples --disable-http"
+      
+
+# native    is for the strange ola_protoc_plugin required at build time
+EXTRA_OECONF_append_class-native    = " --disable-all-plugins --disable-e133 --disable-python-libs --disable-examples --disable-http"
+
+# target    are the real config options
+EXTRA_OECONF_append_class-target = " --with-ola-protoc-plugin=${STAGING_BINDIR_NATIVE}/ola_protoc_plugin --disable-python-libs\
+  --enable-http     --enable-dummy            --enable-gpio   --enable-opendmx --enable-e131 \
+  --enable-opendmx  --enable-usbdmx           --enable-usbpro --enable-spi     --enable-spidmx \
+  --enable-artnet   --enable-openpixelcontrol --enable-renard --enable-kinet \
+  --enable-nanoleaf --enable-karate           --enable-espnet --enable-milinst --enable-pathport \
+  --enable-sandnet  --enable-shownet          --enable-stageprofi \
+  --enable-e133 \
+  --enable-examples "
+
+# -fvisibility-inlines-hidden breaks stuff
+CXXFLAGS = "${CFLAGS}"
+
+# The code is not Werror safe
+do_configure_prepend() {
```

Comment:
`--disable-fatal-warnings` will do this, without needing to be quite so hacky. Although we try our best to fix the issues, so ideally can you tell us what errors you're getting and we'll try and fix or bypass them before they become critical in future.

Reply:
You mean as an argument for ./configure ?

Reply:
Correct @desert .

But please either comment here, or pastebin the errors and we can try adn fix them properly.